src/main/java/org/broadinstitute/hellbender/tools/picard/sam/FilterReads.java

```diff
@@ -72,7 +72,7 @@ public String toString() {
     @Argument(
             doc = "Create .reads files (for debugging purposes)",
             optional = true)
-    public boolean WRITE_READS_FILES = true;
+    public boolean WRITE_READS_FILES = false;
```

Comment:
I assumed that since its for debugging purposes it should default to false, and that it was an oversight that it was left set to true. I'm happy to set it back though - let me know which polarity you think is best and I'll write the additional test for it.